piccolo_api/crud/serializers.py

```diff
@@ -131,9 +135,10 @@ def create_pydantic_model(
         #######################################################################
 
         params: t.Dict[str, t.Any] = {
-            "default": None if is_optional else ...,
             "nullable": column._meta.null,
         }
+        if not column._meta.required:
+            params.update({"default_factory": column.get_default_value})
```

Comment:
It's taking me a while to get this merged in - I just need to make sure this still works with the Piccolo admin.

In the old code we're setting the default to `None` if `is_optional`.